### PR TITLE
fix(authz): entitlements fqn casing

### LINKED ISSUE #1359: TDF Policy Data Attributes are case sensitive
TDF Policy Data Attributes seem to be case sensitive and do not get normalized to lower case when doing a keyRewrap, and authorization of getDecision. 

If the TDF Policy Data Attributes is created with cases (example being: `https://example.com/attr/Caps/value/Bad`), the request will get denied even if the user is entitled with `https://example.com/attr/caps/value/bad`.

Steps to Reproduce:

1. Encrypt a TDF with an attribute that contains cases like `https://example.com/attr/Caps/value/Bad`. 
2. Entitle a user with that attribute cases by creating a subject mapping, shouldn't matter I believe, and should be normalized down. Tested with OTDFCTL.
3. Decrypt the TDF and the request hitting Platform has the attributes with cases. (I am on an older version of SDK) so it could be possible that the new version of SDK cleans this up. But server-side should catch this in case a user is doing RPC call rather than using the sdk.

The request should be a denied by Platform.

### PR DESCRIPTION
**Pull request by suchak1**
Closes #1359

### CODE REVIEW

**Comment by jakedoublev** [comment]
✅ from me. I was able to test successfully with `otdfctl` and an encrypt/decrypt flow where the encryption was with a mixed case attribute and there were valid entitlements.

**Review by ttschampel — COMMENTED** [review]
Can we add a unit test for this case sensitive flow?

File: service/authorization/authorization.go [hunk 1]
```diff
@@ -425,16 +425,16 @@ func makeValsByFqnsLookup(attrs []*policy.Attribute, subMapsByVal map[string][]*
 	return fqnAttrVals
 }
 
-// makeScopeMap creates a map of attribute value FQNs.
-func makeScopeMap(scope *authorization.ResourceAttribute) map[string]bool {
+// MakeScopeMap creates a map of attribute value FQNs.
+func MakeScopeMap(scope *authorization.ResourceAttribute) map[string]bool {
```
Comment (strantalis) [inline service/authorization/authorization.go:429]: @suchak1  is there a reason for this to be public?

Comment (suchak1) [reply service/authorization/authorization.go]: reverted

**Review by jakedoublev — APPROVED** [review]

**Review by strantalis — APPROVED** [review]

**Pull request merged**